demos/tests/data_sequences.py

```diff
@@ -213,4 +219,12 @@
         image_net_arg('00037128'),
         image_net_arg('00048316'),
     ],
+
+    'brain-tumor-nifti': [
+        brats_arg('BRATS_485.nii.gz'),
+    ],
+
+    'image-retrieval-video': [
+        image_retrieval_arg('4946fb41-9da0-4af7-a858-b443bee6d0f6.dav'),
+    ]
```

Comment:
Please move these so that the list remains in sorted order.

Reply:
done